### PR TITLE
Xunit not junit

### PR DESCRIPTION
**Pull request by trieloff**
The Unit report builder was unused, this removes it

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -50,9 +50,8 @@
     "eslint-plugin-header": "3.0.0",
     "eslint-plugin-import": "2.22.0",
     "jsdoc-to-markdown": "6.0.1",
-    "junit-report-builder": "2.0.0",
     "lint-staged": "10.2.11",
-    "mocha": "7.2.0",
+    "mocha": "8.0.1",
```
Comment (tripodsan) [inline package.json:54]: mocha 8.0.1 is flagged by tidelift

Comment (trieloff) [reply package.json:54]: Hm. Not in this repo.

Comment (tripodsan) [reply package.json:54]: strange

**Pull request merged**